ga/README.en.md

```diff
@@ -0,0 +1,46 @@
+# Parameter search by Genetic Algorithm
+
+[日本語](README.md) | English
+
+## Overview
+
+Contains a set of source code to perform parameter search of the urn-model using the Genetic Algorithm (GA).
```

Comment:
Small thing but I think this sounds more natural:

Contains the source code to perform a parameter search of the urn-model using the Genetic Algorithm (GA).